### PR TITLE
pynagsystemd: init at 1.2.0 and dependencies

### PR DESCRIPTION
**Pull request by symphorien**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/nagiosplugin/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+buildPythonPackage rec {
+  pname = "nagiosplugin";
+  version = "1.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vr3zy0zfvbrqc4nf81zxv4gs2q82sv5sjamdm4573ld529mk2nv";
+  };
+
+  buildInputs = [ twine ];
```
Comment (FRidh) [inline pkgs/development/python-modules/nagiosplugin/default.nix]: How is twine used? We basically never add Python modules as `buildInputs`.

Comment (symphorien) [reply pkgs/development/python-modules/nagiosplugin/default.nix]: it is a build time dependency only: https://github.com/mpounsett/nagiosplugin/blob/master/requirements_build.txt

Comment (jonringer) [inline pkgs/development/python-modules/nagiosplugin/default.nix]:
```suggestion
  nativeBuildInputs = [ twine ];
```

File: pkgs/development/python-modules/nagiosplugin/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  checkInputs = [ pytest numpy ];
+
+  doCheck = true;
```
Comment (jonringer) [inline pkgs/development/python-modules/nagiosplugin/default.nix]: this is the default with buildPythonPackage
```suggestion
```

**Comment by jonringer** [comment]
please squash the commits into the related package commit

**Comment by symphorien** [comment]
done

**Review by jonringer — APPROVED** [review]
diff LGTM
commits LGTM
has tests :+1: 

```
[6 built, 113 copied (514.2 MiB), 118.4 MiB DL]
https://github.com/NixOS/nixpkgs/pull/76641
3 package were built:
pynagsystemd python37Packages.nagiosplugin python38Packages.nagiosplugin
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build pynagsystemd python37Packages.nagiosplugin python38Packages.nagiosplugin

**Comment by jonringer** [comment]
arm failures unrelated to PR

**Pull request merged**